pkg/hubble/observer/types/types.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
what's the tldr on this full header vs SPDX in the cilium repo?

Reply:
I don't think we've done the switch for Go source files yet. C and Makefiles do have it however.

Reply:
```
❯ rg SPDX -g '*.go' | grep -v vendor | wc -l
     346
```